packages/react-app/src/helpers/Transactor.js

```diff
@@ -118,11 +118,16 @@ export default function Transactor(providerOrSigner, gasPrice, etherscan) {
 
         return result;
       } catch (e) {
-        console.log(e);
-        console.log("Transaction Error:", e.message);
+        let message;
+        if(e.data) // Metamask signer returns data 
+          message = e.data.message;
+        else // Ethers js needs parsing the error
+          message = JSON.parse(JSON.parse(JSON.stringify(e.error)).body).error.message;
```

Comment:
This throws on Rinkeby, seems like the error object has a different shape 
![image](https://user-images.githubusercontent.com/9612972/124039844-a00a9980-d9fb-11eb-8c7f-aa84554cec0b.png)
Maybe if it doesn't have this expected shape we can just print the whole slug? (as I feel like there might be a few variations, based on type of error message...)

Reply:
My bad this is still valid, I had switched back to Master